components/more_direct_channels/list_item/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import ListItem from './list_item';
+
+export default ListItem;
```

Comment:
I thought we might originally, but I didn't end up adding one. It might make sense in the future since I'm not a huge fan of attaching `last_post_at` to UserProfiles and `profiles` to group Channels, but I didn't end up doing that at the time.

Reply:
Makes sense. Do you think we'll get to that or should we just remove it for now?

Reply:
I think we can leave it. I'm not really opposed to having empty index files to avoid moving stuff around when we add/remove redux logic from components.